lib/ruby-pg-extras/queries/buffercache_usage.sql

```diff
@@ -0,0 +1,7 @@
+select c.relname,count(*) as buffers
+from pg_class c
+inner join pg_buffercache b on b.relfilenode = c.relfilenode
+inner join pg_database d on (b.reldatabase = d.oid and d.datname = current_database())
+group by c.relname
+order by 2 desc
```

Comment:
Why order by 2?

Reply:
`2` refers to the second column of `select` clause.